Client/Application/SceneDelegate.swift

```diff
@@ -134,5 +134,23 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                                               tabSetting: NewTabAccessors.getNewTabPage(profile.prefs)) {
             sceneCoordinator?.findAndHandle(route: route)
         }
+
+        // Check if our connection options include a user response to a push
+        // notification that is for Sent Tabs. If so, route the related tab URLs.
+        if let notification = connectionOptions.notificationResponse?.notification,
+           let userInfo = notification.request.content.userInfo["sentTabs"] as? [[String: Any]] {
```

Comment:
Wondering if we should have a constant for `"sentTabs"` somewhere?

Reply:
@lmarceau Sorry for the delayed reply! Yes I was wondering about that also, and I was curious if the application services team may have a canonical document that lists all of the various APN payloads? It seems like these should perhaps be documented a bit better but I don't have a ton of context with this part of the codebase.

Reply:
@lmarceau So based on feedback in app-services it sounds like there isn't really a canonical document where this payload is defined (apart from the code itself). I think at some point it would be nice to eliminate the use of  these string literals (here also, but mostly `displayNewSentTabNotification()`) but since that's a separate task and not really related to this bug fix I'm thinking it would be better to address that in a follow-up refactor PR. Let me know if that seems reasonable to you.

Reply:
LGTM, thanks for investigating this 🙏